### PR TITLE
Fix bug day off admin page

### PR DESCRIPTION
**Pull request by khanhnguyen1025**

### CODE REVIEW

File: src/app/pages/dayoff/dayoff.component.html [hunk 1]
```diff
@@ -1,54 +1,86 @@
...
+  <nz-table
+    [nzFrontPagination]="false"
+    [nzShowPagination]="false"
+    #nzTable
+    [nzData]="data"
+  >
+    <thead>
+      <tr>
+        <th>ID</th>
+        <th>Day Start Off</th>
```
Comment (thanglenova) [inline src/app/pages/dayoff/dayoff.component.html]: please rename to Start day off

File: src/app/pages/dayoff/dayoff.component.html [hunk 2]
```diff
@@ -1,54 +1,86 @@
...
+        <th>Day End Off</th>
```
Comment (thanglenova) [inline src/app/pages/dayoff/dayoff.component.html]: Please rename to End day off

File: src/app/pages/dayoff/dayoff.component.html (hunk 1 above)
Comment (khanhnguyen1025) [reply src/app/pages/dayoff/dayoff.component.html]: done

File: src/app/pages/dayoff/dayoff.component.html (hunk 2 above)
Comment (khanhnguyen1025) [reply src/app/pages/dayoff/dayoff.component.html]: done

**Pull request merged**